codegenerator/cli/src/persisted_state/db.rs

```diff
@@ -70,15 +77,17 @@ impl PersistedStateExists {
     pub async fn read_from_db_with_pool(
         pool: &PgPool,
     ) -> Result<PersistedStateExists, sqlx::Error> {
-        let val = sqlx::query_as::<_, PersistedState>(
+        let public_schema = get_env_with_default("ENVIO_PG_PUBLIC_SCHEMA", "public");
```

Comment:
Probably should use EnvState here as well.

Reply:
Done